Python/omevv/v1/scripts/CreateRepoProfile.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import sys
+import time
+import warnings
+from webbrowser import get
+
+import requests,json,base64
+warnings.filterwarnings("ignore")
+
+parser = argparse.ArgumentParser(
+    description="Python script using OME api to create Repository profile"
+);
+parser.add_argument('-ip', help='OMEVV IP address', required=True);
+parser.add_argument('-ome_user', help='OME username', required=True);
+parser.add_argument('-ome_pswd', help='OME password', required=True);
+parser.add_argument('-omevv_user', help='OMEVV username', required=True);
+parser.add_argument('-omevv_pswd', help='OMEVV password', required=True);
+parser.add_argument('-profilename', help='repository profile name', required=True);
+parser.add_argument('-description', help='repository profile description', required=False);
+parser.add_argument('-profileType', help='repository profile type', required=True);
+parser.add_argument('-sharepath', help='sharepath', required=True);
+parser.add_argument('-spcred', help='sharepath credential', required=True);
+parser.add_argument('-protocol',help='protocol type',choices=['CIFS','NFS','HTTP','HTTPS','UNAVAILABLE'],required = True);
+parser.add_argument('-certificateCheck',help ='certificate check required?',required = False);
+
+args = vars(parser.parse_args())
+ome_ip = args['ip'];
+ome_user_name = args['ome_user'];
+ome_cred = args['ome_pswd'];
+omevv_user_name = args['omevv_user'];
+omevv_cred = args['omevv_pswd'];
+retry = 3;
+payload ={};
+
+def create_payload():
+    if args["profilename"]:
+        payload["profileName"] = args["profilename"];
+    if args["description"] :
+       payload["description"] = args["description"];
+    if args["profileType"]:
+        payload["profileType"] = args["profileType"];
+    if args["sharepath"]:
```

Comment:
Also we must attempt to validate the format of the sharepath as well based on the protocol.